app/views/devise/sessions/new.html.erb

```diff
@@ -1,20 +1,36 @@
-<h2>Log in</h2>
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12.col-sm-3.col-sm-offset-4">
+      <div class="form-signup">
 
-<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name), data: { turbo: :false }) do |f| %>
-  <div class="form-inputs">
-    <%= f.input :username,
-                required: false,
-                autofocus: true,
-                input_html: { autocomplete: "username" } %>
-    <%= f.input :password,
-                required: false,
-                input_html: { autocomplete: "current-password" } %>
-    <%= f.input :remember_me, as: :boolean if devise_mapping.rememberable? %>
-  </div>
+      <h2>Sign in</h2>
 
-  <div class="form-actions">
-    <%= f.button :submit, "Log in" %>
-  </div>
-<% end %>
+        <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name), data: { turbo: :false }) do |f| %>
+          <div class="form-inputs">
+            <%= f.input :username,
+                        required: false,
+                        autofocus: true,
+                        input_html: { autocomplete: "username" },
+                        placeholder: "alex" %>
+            <%= f.input :password,
+                        required: false,
+                        input_html: { autocomplete: "current-password" },
+                        placeholder: "......" %>
+            <%= f.input :remember_me, as: :boolean if devise_mapping.rememberable? %>
+          </div>
 
-<%= render "devise/shared/links" %>
+          <div class="form-actions text-center">
+            <%= f.button :submit, "Sign in", class:"btn-primary btn-action"%>
+            <br>
+            <br>
+            <%= f.button :submit, "Create Account", class:"btn-outline-primary btn-action"%>
```

Comment:
Remove this and apply the style in app/views/devise/shared/_links.html.erb

Reply:
added them in!